mmv1/products/metastore/Service.yaml

```diff
@@ -184,6 +200,32 @@ properties:
       - :DEVELOPER
       - :ENTERPRISE
     default_from_api: true
+    conflicts:
+      - scalingConfig
+
+  - !ruby/object:Api::Type::NestedObject
+    name: 'scalingConfig'
+    description: |
+      Represents the scaling configuration of a metastore service.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'instanceSize'
+        description: |
+          Metastore instance sizes.
+        required: false
+        conflicts:
+          - tier
```

Comment:
This conflicts with the top-level field `tier`?

Reply:
Got confirmation from @nitikhadapkar that this is correct.